vocab/security/vocabulary.yml

```diff
@@ -58,6 +58,10 @@ class:
     upper_value: sec:Proof
     defined_by: https://www.w3.org/TR/vc-di-eddsa/#ed25519signature2020
 
+  - id: Errors
+    label: Processing errors
+    defined_by: https://www.w3.org/TR/vc-di-eddsa/#errors
+
 
 # These are the class definitions in the CCG documents that are not defined in a VCWG document; they are all deprecated
 # In some cases a ccg document was found and used for the definition, but in some cases even that is missing...
```

Comment:
```suggestion
# These are the class definitions in the CCG documents that are not defined in a VCWG document; they are all deprecated.
# In some cases, a CCG document was found and used for the definition, but in other cases, even that is missing...
```

Reply:
@TallTed I will merge this but... this is only a comment in the YAML file...